recipe-generator-node/download-generator-jar.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const https = require("https");
+
+const { FILEPATH, FILE_URL, OUTPUT_DIR, VERSION } = require("./vars");
+
+if (!fs.existsSync(OUTPUT_DIR)) {
+    fs.mkdirSync(OUTPUT_DIR);
```

Comment:
I guess we know `OUTPUT_DIR` is a direct child directory of the current directory so my previous comment about maybe needing `mkdir -p` isn't applicable anymore?

Reply:
Yup